tests/test_wog.py

```diff
@@ -140,6 +140,20 @@ def test_chogall_cannot_pay_health():
 	assert not fireball.is_playable()
 
 
+def test_cult_apothecary():
+	game = prepare_game()
+	cult_apothecary = game.player1.give("OG_295")
+	game.end_turn()
+	game.player2.give(WISP).play()
+	game.player2.give(WISP).play()
+	game.player2.give(WISP).play()
+	game.player2.give(WISP).play()
+	game.player2.give("EX1_279").play(target=game.player1.hero)
+	game.end_turn()
+	cult_apothecary.play()
+	assert game.player1.hero.health == 28
```

Comment:
Add a lightwarden to this test so you can double check the heal only triggers once.